src/components/entry/Entry.jsx

```diff
@@ -1,14 +1,14 @@
 import { StyledEntry } from './Entry.styled';
 import Typography from '../ui/Typography';
 
-export default function Entry({ entry, location }) {
+export default function Entry({ entry }) {
 	return (
 		<StyledEntry>
 			<Typography variant="h3">Name:</Typography>
 			<Typography variant="body1">{entry.name}</Typography>
-			<Typography>Place</Typography>
-			<Typography variant="body1">long: {location.location[0]}</Typography>
-			<Typography variant="body1">lat: {location.location[1]}</Typography>
+			<Typography variant="h3">Place:</Typography>
+			<Typography variant="body1">long: {entry.location[0]}</Typography>
+			<Typography variant="body1">lat: {entry.location[1]}</Typography>
```

Comment:
Instead of using your Typography component twice for long and lat, you could use a gold old `<br/>` and put those two values within one Typography component with variant="body1"